stablehlo/conversions/linalg/transforms/StablehloLegalizeToLinalg.cpp

```diff
@@ -818,7 +818,7 @@ struct TransposeConverter final
     SmallVector<AffineExpr, 2> inputExprs;
     inputExprs.resize(resultType.getRank());
     for (auto [idx, value] : llvm::enumerate(op.getPermutation())) {
-      inputExprs[value.getZExtValue()] = b->getAffineDimExpr(idx);
```

Comment:
Any idea if this change is related to using dense array?

Reply:
Yeah, because permutation is now an array of i64, looping over it uses i64 instead of APInt and we no longer need the conversion to i64.